README.md

```diff
@@ -1 +1,61 @@
-# idea-Box
\ No newline at end of file
+# idea-Box
+
+### `Abstract:`
+[//]: <> (Briefly describe what you built and its features. What problem is the app solving? How does this application solve that problem?)
+Every developer has more ideas than time. As David Allen likes to say “the human brain is for creating ideas, not remembering them.” In this project, we’ll be building an application that records and archives our ideas (good and bad alike).
+
+Throughout the project, one of our focuses will be on providing a fluid and responsive client-side interface. To this end, we’ll rely on JavaScript to implement snappy filtering in the browser.
```

Comment:
Do we want to add anything about our localStorage feature so ideas are not lost on refresh? I feel like that is huge!